ingenialogger/ingenialogger.py

```diff
@@ -96,23 +89,49 @@ def configure_logger(level=logging.WARNING, queue=False, file=None):
     if not check_logger_handler(root_logger, ingenia_handlers.steam_handler):
         root_logger.addHandler(ingenia_handlers.steam_handler)
 
-    if file:
-        if ingenia_handlers.file_handler is None:
-            file_handler = logging.FileHandler(file)
-            file_handler.setFormatter(formatter)
-            ingenia_handlers.file_handler = file_handler
-        if not check_logger_handler(root_logger, ingenia_handlers.file_handler):
-            root_logger.addHandler(ingenia_handlers.file_handler)
-
-    if queue:
-        if ingenia_handlers.queue_handler is None:
-            log_queue = Queue()
-            queue_handler = QueueHandler(log_queue)
-            queue_handler.setFormatter(formatter)
-            ingenia_handlers.queue_handler = queue_handler
-        if not check_logger_handler(root_logger, ingenia_handlers.queue_handler):
-            root_logger.addHandler(ingenia_handlers.queue_handler)
-        return ingenia_handlers.queue_handler.queue
+
+def configure_file_handler(filename):
+    """Configure a FileHandler.
+
+    Args:
+        filename: Path to the file where the log will be stored.
+    """
+
+    root_logger = logging.getLogger()
+    formatter = logging.Formatter(FORMAT)
+    print(ingenia_handlers)
+    if ingenia_handlers.file_handler is None:
+        file_handler = logging.FileHandler(filename)
+        file_handler.setFormatter(formatter)
+        ingenia_handlers.file_handler = file_handler
+    if not check_logger_handler(root_logger, ingenia_handlers.file_handler):
+        root_logger.addHandler(ingenia_handlers.file_handler)
+
+
+def configure_queue_handler():
+    """Configure a QueueHandler.
+
+    Returns:
+        queue.Queue: Log queue.
```

Comment:
I think it's like this:
`Queue: Log queue`

Reply:
If only Queue is written, documentation has no link to the python documentation
![image](https://user-images.githubusercontent.com/3721567/225376260-60030cca-e050-43ab-9c27-8219fd175eac.png)